pkg/lib/serverTime.js

```diff
@@ -496,6 +560,20 @@ function ChangeSystimeBody({ state, errors, change }) {
             </tbody>
         </table>);
 
+    const mode_options = [
+        <SelectOption key="1" value="manual_time">{_("Manually")}</SelectOption>,
+        <SelectOption key="2" isDisabled={!ntp_supported} value="ntp_time">{_("Automatically using NTP")}</SelectOption>
```

Comment:
I suggest using the same values for key= as for value=, it's a bit confusing/error prone otherwise.

Reply:
Yep.